lib/opensusebasetest.pm

```diff
@@ -306,6 +306,10 @@ sub wait_boot {
         if (get_var('ONLINE_MIGRATION')) {
             push @tags, 'migration-source-system-grub2';
         }
+
+        # enter password before grub when FULL_LVM_ENCRYPT=1
+        workaround_type_encrypted_passphrase;
```

Comment:
Yeah, the function is required not only for FULL_LVM_ENCRYPT=1, so my comment is confusing here. And seems the position of inserted function is also possible to cause problem. I will try to change it.
Thanks for your comment.